data/json/items/armor/coats.json

```diff
@@ -734,8 +1102,20 @@
       { "covers": [ "arm_l", "arm_r" ], "coverage": 90, "encumbrance": [ 15, 15 ] }
     ],
     "pocket_data": [
-      { "pocket_type": "CONTAINER", "max_contains_volume": "700 ml", "max_contains_weight": "3 kg", "moves": 80 },
-      { "pocket_type": "CONTAINER", "max_contains_volume": "700 ml", "max_contains_weight": "3 kg", "moves": 80 }
+      {
+        "pocket_type": "CONTAINER",
+        "max_contains_volume": "800 ml",
+        "max_contains_weight": "3 kg",
+        "max_item_length": "14 cm",
```

Comment:
Out of curiosity why is this 800ml pocket 14 CM and the pocket on line 1073 above it is 800ml with 13CM max length?  Several places where this type of difference occurs.

Reply:
That's the result of working on this for hours and hours and making copy-paste errors.  Thanks for pointing it out!

Reply:
Just to clarify, some jackets indeed do have just barely longer pockets than others, but items that are the same, but with a color difference should have the same stats.